MUXSDKStats/MUXSDKStatsTests/MUXSDKStatsTests.m

```diff
@@ -275,7 +275,7 @@ - (void) testRenditionChangeEvent {
     [[NSNotificationCenter defaultCenter] postNotificationName:RenditionChangeNotification object:renditionInfo];
     
     NSDictionary *renditionInfoWithoutFrameRate = @{
-        RenditionChangeNotificationInfoAdvertisedBitrate: @(258157)
+        RenditionChangeNotificationInfoAdvertisedBitrate: @(558157)
```

Comment:
Reason for this bitrate number changing in the test? 😄 nbd, jw

Reply:
Good eye 😁  It's because of the change I made here: https://github.com/muxinc/mux-stats-sdk-avplayer/pull/56#discussion_r448753922

Previously, we would emit a data event even if the bitrate had not changed if we got two `RenditionChangeNotification`. Now we only do that if the bitrate has changed since the last time we sent it. So I updated the test to reflect that.

Though now that you bring it up, I should also add a unit test covering the case we get two `RenditionChangeNotification` anyway - I'll add this